pilot/cmd/pilot-agent/main.go

```diff
@@ -150,7 +150,7 @@ var (
 	skipParseTokenEnv = env.RegisterBoolVar("SKIP_PARSE_TOKEN", false,
 		"Skip Parse token to inspect information like expiration time in proxy. This may be possible "+
 			"for example in vm we don't use token to rotate cert.").Get()
-	proxyXDSViaAgent = env.RegisterBoolVar("ISTIO_META_PROXY_XDS_VIA_AGENT", false,
+	proxyXDSViaAgent = env.RegisterBoolVar("PROXY_XDS_VIA_AGENT", true,
```

Comment:
How will people disable it if stuff blows up? Set the env var on all the pods?

Reply:
No you can do it in mesh config (see release notes for example)

Reply:
If there is a chance it can blow up, it should not be enabled by default. Why is it going to blow up?

Reply:
There is always a chance something can blow up. Standard practice has been to ship it disabled, then enabled by default, then (sometimes) always on with no knob.